dandi/urls.py

```diff
@@ -26,9 +29,21 @@
     )
 )
 
+schema_view = get_schema_view(
+    openapi.Info(
+        title='ABC',
+        default_version='v1',
+        description='abc123',
```

Comment:
Maybe `The BRAIN Initiative archive for publishing and sharing cellular neurophysiology data`